src/bots/slackbot.ts

```diff
@@ -82,6 +82,24 @@ export namespace SlackBot {
 		}
 	}
 
+	/**
+	 *  This function registers a command and handles exceptions
+	 *  To not use try/catch in the `cb()` function, it will be caught automatically and a message will be logged and sent to the user
+	 */
+	export function registerCommand(cmd: string, cb: (respond: RespondFn, body: SlashCommand) => Promise<void> | void) {
+		slackApp.command(cmd, async (context) => {
+			// commands should always be acknowledged within 3 seconds
```

Comment:
Could you provide me the URL where this is specified ?

Reply:
https://slack.dev/bolt-js/concepts#commands
> We recommend calling ack() right away before sending a new message or fetching information from your database since you only have 3 seconds to respond.